### DISCUSSION TITLE
Rust SDK

### DISCUSSION THREAD

**Discussion by globalflea** [serverlessworkflow/specification#530] [Q&A]
Is there any plans for a Rust SDK? If so, any timeline?

**Comment by tsurdilo** [comment]
Yes there are plans, no timeline. We will need community to help contribute to it. Interested?

**Reply by globalflea** [reply]
Don't mind, but I am new to Rust myself.

**Reply by tsurdilo** [reply]
wanna help us out on the temporal integration (java)? let me know and ill add you to current repo and slack discussions.

**Reply by globalflea** [reply]
not java, i'm not keen on using java nowadays, even though i had used it in projects years ago.   Would rather practise Go or struggle with Rust.  So i think if you've anything for these two, i wouldn't mind trying to help out where i can

**Reply by ricardozanini** [reply]
We have a GO SDK :) 
Contributions are more than welcome: https://github.com/serverlessworkflow/sdk-go

**Reply by globalflea** [reply]
i think it would be interesting to try to build the workflow spec on top of temporal.io using the go-sdk, esp since temporal itself is built on Go.  i am new to temporal, having been introduced to it by tsurdilo in another discussion.  Also, i am relatively new to Go.  Nonetheless, if there's any effort on this front, i don't mind helping out where i can

**Comment by ricardozanini** [comment]
We still don't have any plans to have a Rust SDK, but of course any contributions are welcome!
